### DISCUSSION TITLE
FAQ link to deployment docs?

### DISCUSSION THREAD

**Discussion by mrlacey** [microsoft/WindowsAppSDK#1314] [Q&A] [answered]
[The FAQs refer to](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/faq.md#do-i-need-to-be-packaged-to-use-the-windows-app-sdk) " we'll have instructions to carry the Windows App SDK components with your app and how to install them using some common installers."

Is that these: https://docs.microsoft.com/windows/apps/windows-app-sdk/deploy-unpackaged-apps  ?

**Comment by btueffers** [comment, marked as answer]
Hi @mrlacey, yes that is the correct page! The FAQs will be updated shortly to include a link, thank you for your feedback!
